viogpu/common/viogpu_queue.h

```diff
@@ -49,8 +49,13 @@ typedef struct virtio_gpu_vbuffer {
 
     char *resp_buf;
     int resp_size;
-    PKEVENT event;
     LIST_ENTRY list_entry;
+
+    void (*complete_cb)(void *ctx);
+    void *complete_ctx;
+
+    bool no_auto_release;
```

Comment:
This is not a good practice to make variable names with negation suffixes. It looks weird and hard to read: `if (!pvbuf->no_auto_release) {`. 

Are this any blockers to change this to `bool auto_release;` and `if (pvbuf->auto_release) {`?

Reply:
Done.